app/controllers/topical_events_controller.rb

```diff
@@ -4,9 +4,9 @@ class TopicalEventsController < ClassificationsController
   def show
     @classification = TopicalEvent.friendly.find(params[:id])
     @content_item = Whitehall.content_store.content_item(@classification.base_path)
-    @publications = fetch_associated(:published_publications, PublicationesquePresenter)
-    @consultations = fetch_associated(:published_consultations, PublicationesquePresenter)
-    @announcements = fetch_associated(:published_announcements, AnnouncementPresenter)
+    @publications = fetch_publications
+    @consultations = fetch_consultations
+    @announcements = fetch_announcements
```

Comment:
This is the meat of the change. Rather than looking up related documents in Whitehall, we're fetching them from Rummager.